oaknational/Oak-Web-Application
# Feat/lesq 801/homepage updates [LESQ-801]

## Description

Music year: 2020

- Update hero images in sanity for pupils and curriculum
- Add hero image for AI
- Update tab icon for Ai to waving robot
- Update copy for keystage selector on teachers' tab

## How to test

1. Go to https://deploy-preview-2414--oak-web-application.netlify.thenational.academy
2. Click on AI tab
3. You should see an illustration, a new robot icon
4. The curriculum and pupils images should have a thicker border matching the teachers tab (this change was done via sanity so it already is on production)
5. Keystage selector text should be updated on the teachers tab but unchanged in the subject listing page

## Screenshots

How it used to look (delete if n/a):  
<img width="500" alt="Screenshot 2024-05-09 at 08 27 20" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/8ea3207c-2d08-425d-a5fc-3ab8c742ace2">
<img width="500" alt="Screenshot 2024-05-09 at 08 33 59" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/93b4eb4a-70e8-47e4-bc7d-9324a959b07c">


How it should now look:  
<img width="500" alt="Screenshot 2024-05-09 at 08 26 46" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/bbcc4f4e-22cf-458c-b75f-b08ade8fd789">
<img width="500" alt="Screenshot 2024-05-09 at 08 34 11" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/b4df737f-d351-4b5c-869c-c78a9f4bbd44">

## ACs

- [x]  Update images in the top area to have consistent border thickness (currently the first one is thicker than others)
- [x]  Replace AI icon in the tabs to use a robot
- [x]  Add an illustration for AI area.
- [x]  Update copy for the Key Stage selector to “ View subjects by key stage”
